src/compositor/output.c

```diff
@@ -600,6 +600,25 @@ wlc_output_set_backend_surface(struct wlc_output *output, struct wlc_backend_sur
    return false;
 }
 
+static void
+wlc_output_update_mode(struct wlc_output *output)
+{
+
+   bool set_resolution = false;
+
+   {
```

Comment:
Oh, I guess you just moved this code from elsewhere. Still, would remove the braces in this case.